src/pages/ModelOfferPage.tsx

```diff
@@ -70,18 +81,23 @@ const ModelOfferPage = () => {
         <S.ModelOfferBox>
           <TitleBox title="상세 제안" subtitle="자세히 적을수록 매칭 성공률이 높아져요" isNeccessary={true} />
           <TextArea200
-            placeholderText="내용을 입력해주세요&#13;&#10;예시) 상세 가격조건, 구체적인 스타일 제안, 시술 시간 등`"
+            placeholderText="내용을 입력해주세요 &#13;&#10;예시) 상세 가격조건, 구체적인 스타일 제안, 시술 시간 등"
             onChangeFn={handleTextAreaChange}
           />
         </S.ModelOfferBox>
       </S.ModelOfferLayout>
-      <Button
-        id="ga-offer-complete-btn"
-        text="완료하기"
-        isFixed={true}
-        onClickFn={handleClickConfirm}
-        disabled={!isActive}
-      />
+      <Button id="ga-offer-complete-btn" text="완료" isFixed={true} onClickFn={handleOpenModal} disabled={!isActive} />
+
+      {isModalOpen && (
+        <Modal
+          title="제안서를 전송할까요?"
+          description="제안한 모델에게는<br/>문자를 통해 알림이 가요"
+          leftBtnFn={handleCloseModal}
```

Comment:
```suggestion
          leftBtnFn={setIsModalOpen(false)}
```
아님 이렇게 바로넣어주는 방법은?! (내가 이렇게 마니 해서..)
하지만 이건 개인취향이니 민서 편한대루 .. 🖤🖤